tests/baselines/reference/getEmitOutputSingleFile2.baseline

```diff
@@ -23,7 +23,7 @@ declare class Foo {
     x: string;
     y: number;
 }
-declare module "inputFile3" {
+declare module "inputfile3" {

```

Comment:
This is as a result of the canonicalization now performed on the file name in order to get the whole path (and then trim off the common part).

I suppose I could rework the comparison so not canonicalize and split, but instead compare canonicalized components and join, in order to preserve input casing on case insensitive systems?